bucket/winssh-pageant.json

```
@@ -0,0 +1,42 @@
+{
+    "version": "1.2",
+    "description": "Pageant to Windows OpenSSH agent bridge",
+    "homepage": "https://github.com/ndbeals/winssh-pageant",
+    "license": "BSD-3-Clause",
+    "notes": [
+        "This application is meant to run in the background upon system startup. Add a shortcut to %APPDATA%\\Microsoft\\Windows\\Start Menu\\Startup that points to $dir\\winssh-pageant.exe",
+        "Alternatively, you can run the following snippet in powershell:",
+        "$Shortcut = (New-Object -comObject WScript.Shell).CreateShortcut($env:APPDATA+'\\Microsoft\\Windows\\Start Menu\\Programs\\Startup\\winssh-pageant.lnk')","$Shortcut.TargetPath = '$dir\\winssh-pageant.exe'",
+        "$Shortcut.Save()"
+    ],
```

It is not good idea to use %appdata% as it is cmd only and it ignores global installation options

```suggestion
    "notes": "This application is meant to run in the background upon system startup. If you want to fully use it, copy shortcut into Start Menu Startup folder",
```

---

> it ignores global installation options

That is the intention. This is a tool that works with the users ssh-agent. Adding this as a startup item to other users that may not even use an ssh-agent is wrong.

---

Theses intentions are nowhere mentioned. It is not handled. Noone will know this.

---

Anyone using an ssh-agent (for which this tool only works for) will know this by default. That is how ssh-agent works. The intent of this tool, to be a bridge to the windows ssh agent is in the description. I have updated the notes anyways.